frontends/concrete-python/tests/execution/test_others.py

```diff
@@ -966,6 +966,14 @@ def issue650(x):
             {},
             id="issue-651",
         ),
+        pytest.param(
```

Comment:
where is the new option tested ?

Reply:
ah ok it test the default behaviour is that this bug example compiles.
maybe a test showing the optimization still works if not already somewhere.

Reply:
Yeap, it was testing the default behavior, but I've added more tests to test every case now.